### PR TITLE
Update land-sea mask

### LINKED ISSUE #173: Update the land-sea mask
The land-sea mask should be updated to use the mask here: [https://www.ncl.ucar.edu/Applications/Data/#cd](https://www.ncl.ucar.edu/Applications/Data/#cd).

### PR DESCRIPTION
**Pull request by ph-kev**
closes #173, #177 - This PR updates the land-sea mask to use the mask found here: [https://www.ncl.ucar.edu/Applications/Data/#cd](https://www.ncl.ucar.edu/Applications/Data/#cd).

The plot now look like this with an ocean mask:
plot_bias_oceanmask

Notice that the boarder is not completely shaded in because the mask is only at 1 degree resolution.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -23,6 +24,7 @@ ClimaAnalysisMakieExt = "Makie"
 
 [compat]
 Aqua = "0.8"
+Artifacts = "1.9.4"
```
Comment (Sbozzolo) [inline Project.toml:27]:
```suggestion
Artifacts = "1"
```

File: .github/workflows/Downgrade.yml [hunk 2]
```diff
@@ -29,6 +29,6 @@ jobs:
       - uses: julia-actions/cache@v2
       - uses: julia-actions/julia-downgrade-compat@v1
         with:
-          skip: Dates, Test, Statistics
+          skip: Dates, Test, Statistics, Artifacts
```
Comment (Sbozzolo) [inline .github/workflows/Downgrade.yml:32]:
```suggestion
          skip: Artifacts, Dates, Test, Statistics
```

File: NEWS.md [hunk 3]
```diff
@@ -38,6 +38,18 @@ below.
...
 
+### Converting the units of a dimension
+Similar to converting the units of data, you can convert the units of a dimension with
+`ClimaAnalysis.convert_dim_units`. See the example below.
+```julia
+new_var = ClimaAnalysis.convert_dim_units(
+        var,
+        "lat",
+        "rads",
+        conversion_function = x -> x * π / 180.0,
```
Comment (Sbozzolo) [inline NEWS.md:49]:
```suggestion
        conversion_function = x -> x * π / 180,
```

File: docs/src/var.md [hunk 4]
```diff
@@ -55,20 +55,35 @@ new_var = ClimaAnalysis.convert_units(var, "kg m/s", conversion_function = (x) -
...
-Similarly, to set the units of a dimension, one can use the `dim_set_units!` function.
+For converting the units of a dimension, you can use
+[`ClimaAnalysis.convert_dim_units`](@ref). As of now, automatic conversion is not supported
+which means you need to supply the conversion function. See the example below.
+```julia
+new_var = ClimaAnalysis.convert_dim_units(
+        var,
+        "lat",
+        "rads",
+        conversion_function = x -> x * π / 180.0,
```
Comment (Sbozzolo) [inline docs/src/var.md:73]:
```suggestion
        conversion_function = x -> x * π / 180,
```

File: src/constants.jl [hunk 5]
```diff
@@ -1,23 +1,53 @@
...
-else.
+const LAND_MASK = begin
+    path = joinpath(
+        Artifacts.artifact"landsea_mask_30arcseconds",
+        "landsea_mask.nc",
+    )
+    ds = NCDatasets.NCDataset(path)
+    lon = ds["lon"][begin:16:end]
+    lat = ds["lat"][begin:16:end]
+    data = ds["landsea"][begin:16:end, begin:16:end]
```
Comment (Sbozzolo) [inline src/constants.jl:17]: Do you need to take 1 element every 16th?

File: src/constants.jl [hunk 6]
```diff
@@ -1,23 +1,53 @@
...
+"""
+const OCEAN_MASK = begin
+    path = joinpath(
+        Artifacts.artifact"landsea_mask_30arcseconds",
+        "landsea_mask.nc",
+    )
+    ds = NCDatasets.NCDataset(path)
+    lon = ds["lon"][begin:16:end]
+    lat = ds["lat"][begin:16:end]
+    data = ds["landsea"][begin:16:end, begin:16:end]
```
Comment (Sbozzolo) [inline src/constants.jl:42]: Same here

File: NEWS.md (hunk 3 above)
Comment (ph-kev) [reply NEWS.md]: This commit is already in main. I am not sure if I should make another commit to fix these or just leave as it.

File: src/constants.jl (hunk 5 above)
Comment (ph-kev) [reply src/constants.jl:17]: I changed it to taking every 4th element. As a result, the OutputVar is 1 GB in size. For comparison, taking every 8th element is about 0.25 GB.

**Review by Sbozzolo — APPROVED** [review]

**Pull request merged**